app/models/card.rb

```diff
@@ -1,19 +1,38 @@
 class Card < ActiveRecord::Base
-  before_validation :add_review_time
-  
+  scope :rand_word, -> { order("RANDOM()").limit(1) }
+  scope :need_check, -> { where('review_date <= ?', Time.zone.now) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.